.github/labeler.yml

```diff
@@ -1,58 +1,34 @@
-# Add/remove 'ep:ACL' label if issue contains the word 'ACL'
-ep:ACL:
-  - '/ACL/i'
-# Add/remove 'ep:ArmNN' label if issue contains the word 'ArmNN'
-ep:ArmNN:
-  - '(ArmNN)'
-# Add/remove 'ep:CUDA' label if issue contains the word 'CUDA'
-ep:CUDA:
-  - '(CUDA)'
-# Add/remove 'ep:DML' label if issue contains the word 'DirectML'
+ep:ACL: '/\bacl\b/i'
+ep:ArmNN: '/\barmnn\b/i'
+ep:CUDA: '/\bcuda\b/i'
 ep:DML:
-  - '(DirectML)'
-# Add/remove 'ep:MIGraphX' label if issue contains the word 'MIGraphX'
-ep:MIGraphX:
-  - '(MIGraphX)'
-# Add/remove 'ep:oneDNN' label if issue contains the word 'oneDNN'
-ep:oneDNN:
-  - '(oneDNN)'
-# Add/remove 'ep:OpenVINO' label if issue contains the word 'OpenVINO'
-ep:OpenVINO:
-  - '(OpenVINO)'
-# Add/remove 'ep:RockchipNPU' label if issue contains the word 'Rockchip NPU'
-ep:RockchipNPU:
-  - '(Rockchip NPU)'
-# Add/remove 'ep:TensorRT' label if issue contains the word 'TensorRT'
+  - '/\bdirectml\b/i'
+  - '/\bdml\b/i'
+ep:MIGraphX: /\bmigraphx\b/i'
+ep:oneDNN: '/\bonednn\b/i'
+ep:OpenVINO: '/\bopen\s*vino\b/i'
+ep:RockchipNPU: '/\brockchip\b/i'
 ep:TensorRT:
-  - '(TensorRT)'
-# Add/remove 'ep:tvm' label if issue contains the word 'TVM'
-ep:tvm:
-  - '(TVM)'
-# Add/remove 'ep:VitisAI' label if issue contains the word 'Vitis AI'
-ep:VitisAI:
-  - '(Vitis AI)'
-# Add/remove 'api:JavaScript' label if issue contains the word 'JavaScript'
-api:JavaScript:
-  - '(JavaScript)'
-# Add/remove 'api:Java' label if issue contains the word 'Java' not followed by 'Script'
-api:Java:
-  - '(Java(?!Script))'
-# Add/remove 'api' label if issue contains the word 'C#'
-api:CSharp:
-  - '(C#)'
-# Add/remove 'api' label if issue contains the word 'Python', 'C++', 'C\b', 'Obj-C', or 'WinRT'
-api:
-  - '(Python|C\+\+|C\b|Obj-C)'
-# Add/remove 'platform:jetson' label if issue contains the word 'Jetson' or 'jetson'
-platform:jetson:
-  - '(Jetson|jetson)'
-# Add/remove 'platform:windows' label if issue contains the word 'Windows'
+  - '/\btensor\s*rt\b/i'
+  - '/\btrt\b/i'
+ep:tvm: '/\btvm\b/i'
+ep:VitisAI: '/\bvitus(?:ai)?\b/i'
+platform:jetson: '/\bjetson\b/i'
+platform:mobile:
+  - '/\bobj-?c\b/i'
```

Comment:
can we also handle "objective-c" and "objectivec"?

Reply:
Added